README.md

```diff
@@ -1,10 +1,10 @@
-[<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="400"/>][homepage]
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="400"/>](https://mineiros.io/?ref=terraform-aws-cognito-user-pool)
 
-[![Build Status][badge-build]][build-status]
-[![GitHub tag (latest SemVer)][badge-semver]][releases-github]
-[![Terraform Version][badge-terraform]][releases-terraform]
-[![AWS Provider Version][badge-tf-aws]][releases-aws-provider]
-[![Join Slack][badge-slack]][slack]
+[![Build Status](https://github.com/mineiros-io/terraform-aws-cognito-user-pool/workflows/Tests/badge.svg)](https://github.com/mineiros-io/terraform-aws-cognito-user-pool/actions)
```

Comment:
Old image URL was [this](https://github.com/mineiros-io/terraform-aws-lambda-function/workflows/CI/CD%20Pipeline/badge.svg). New one (following template layout) doesn't seem to exist. How can I fix this?

Reply:
hey @kassianh, the URL of the badge is formed using the GH workflow name as part of it, like `https://github.com/mineiros-io/terraform-aws-cognito-user-pool/workflows/<WORKFLOW_NAME>/badge.svg`.

in the majority of repos, we have the workflow name set to `Tests` but in this case the name is `CI/CD Pipeline` as you can see in https://github.com/mineiros-io/terraform-aws-cognito-user-pool/actions/workflows/main.yml

Reply:
Thank you! This makes sense.